src/api/store/userprofile.py

```diff
@@ -37,8 +38,8 @@ def _has_access(self, context: Context, user_id=None, email=None):
 
   def get_user_info(self, context: Context, args) -> (dict, MassEnergizeAPIError):
     try:
-      email = args.get('email', None)
-      user_id = args.get('user_id', None)
+      #email = args.get('email', None)
```

Comment:
why are you commenting these out?

Reply:
Didn't need to.  It fixed the warning from the linter that those variables weren't used.  I think you had commented out the call to self._has_access, which used those variables, but since you left that comment in there I left the commented out variables in there too.